aloha/real_env.py

```diff
@@ -136,33 +127,56 @@ def setup_robots(self):
         setup_follower_bot(self.follower_bot_right)
 
     def get_qpos(self):
-        left_qpos_raw = self.recorder_left.qpos
-        right_qpos_raw = self.recorder_right.qpos
-        left_arm_qpos = left_qpos_raw[:6]
-        right_arm_qpos = right_qpos_raw[:6]
-        left_gripper_qpos = [FOLLOWER_GRIPPER_POSITION_NORMALIZE_FN(left_qpos_raw[7])]
-        right_gripper_qpos = [FOLLOWER_GRIPPER_POSITION_NORMALIZE_FN(right_qpos_raw[7])]
-        return np.concatenate(
-            [left_arm_qpos, left_gripper_qpos, right_arm_qpos, right_gripper_qpos]
-        )
+        # Initialize a list to hold the arm and gripper positions
+        qpos_list = []
+
+        # Iterate through all follower robots in the self.robots dictionary
+        for name, bot in self.robots.items():
```

Comment:
We should document that the order of the observations relies on the order of the arms. We should also make sure that we're backwards compatible with the previous version of aloha - that we're saving the observations in the same order as before for all robot styles.